proxy/src/proxy.rs

```diff
@@ -67,7 +73,7 @@ static NUM_BYTES_PROXIED_COUNTER: Lazy<IntCounterVec> = Lazy::new(|| {
     register_int_counter_vec!(
         "proxy_io_bytes_per_client",
         "Number of bytes sent/received between client and backend.",
-        crate::console::messages::MetricsAuxInfo::TRAFFIC_LABELS,
+        &["direction"],
```

Comment:
I would be careful with removing endpoint_id and other labels from this metric, we may have some places that use this metric.

Quick search for `proxy_io_bytes_per_client` in slack found several mentions in #data and #data-infra-alerts, and we also can have some dashboards relying on it.

We may want to leave those metrics until we're sure they are not used anywhere, or create a config option to be able to enable/disable them and observe what breaks.

Reply:
getting rid of these metrics is the entire reason I want to get this change in 😩

Reply:
> Quick search for proxy_io_bytes_per_client in slack

That might not be for the prom metrics as we use the same key for the billing metrics

Reply:
Yeah, I don't know where and how metrics are used exactly, but feel like people will come with questions after metrics will be removed :) I'd ask in slack if any important pipelines still use these metrics, in #infra and #data.